### ISSUE TITLE
Combine with efforts of SWISS-MODEL team

### ISSUE THREAD

**Issue by gtauriello** [GHDDI-AILab/Targeting2019-nCoV#34]
I noticed with great interest that you have built homology models for SARS-CoV-2 (aka 2019-nCoV) proteins using our SWISS-MODEL service. I would like to use this "feature request" to point you to our own dedicated page for modelling attempts [here](https://swissmodel.expasy.org/repository/species/2697049).

Similarly to your efforts, we envision the page to help life science researchers to get the structural data needed for their further analysis. We plan to keep the page up-to-date by including any additional useful experimental structure that appears.

So I would be happy if we can combine our efforts to provide up-to-date structural data to aid clinical researchers in their efforts to design new therapies.

**Comment by wingring47** [comment]
Hi Gerardo!
Thank you for your contact! Sure we can definitely combine our structural data and thank you very much for your extensive effort in combating the pandemic. 

we can provide your link with some description on our structure page and people can link into it. Would that work? 

Or do you have other suggestion? You can also edit based on our current .md file but that might require some extensive editing. 

Please let me know. 

Thank you!

-Lurong

**Comment by gtauriello** [comment]
Yes the link with the description works well. Thank you for adding it. Just one pedantic comment: could you use "SWISS-MODEL" instead of "Swiss Model"?

Also please let us know if we can provide anything else to your very valuable resource.

Regards,
Gerardo

**Comment by wingring47** [comment]
Sure! We will make the correction. If you have more bioinformation regarding vaccine and antibody development and host immune related bioinformatics against previous corona virus that would be really appreciated. 

Thank you
